java/com/engflow/binaryinput/BUILD

```diff
@@ -0,0 +1,32 @@
+load("@rules_java//java:defs.bzl", "java_binary", "java_library")
+
+NUM_FILES = 1
+
+#Generates a number of java files based on the value of NUM_FILES
+#Each file is named HelloX.java where X is the number of the file
+#Each file contains a class with a greetNum method that prints "Hello" + the number of the file
+[genrule(
```

Comment:
Every comment should have a whitespace just after the symbol `#`. Do that for all comments in the code.